packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/run-transaction-endpoint-v1-serialized.test.ts

```diff
@@ -0,0 +1,310 @@
+import http from "http";
```

Comment:
I think peter was opting for minimizing number of fabric tests to speed up it's CI job, can we push this into already existing test file?

Reply:
@jagpreetsinghsasan Plus one to the above. It's my fault for being inconsistent with the requirements over time, apologies. The performance of the CI is still very much below what we'd like to have in an ideal situation so some shortcuts have to be made and one of them is to accept that some test cases won't run in full isolation (with their dedicated pristine ledger)

Reply:
@outSH @petermetz yes, a much better suggestion. I will add this to an existing test case itself and re-request again.